packages/db-kit/src/cli/decodeTransaction/useTransactionInfo.ts

```diff
@@ -66,18 +66,18 @@ function fetchTransactionInfo({
       const addresses: string[] = [];
 
       // include direct `to`
-      if (transaction.to) {
+      if (transaction?.to) {
         addresses.push(transaction.to);
       }
 
       // or the created contract address for create transactions
-      if (receipt.contractAddress) {
+      if (receipt?.contractAddress) {
         addresses.push(receipt.contractAddress);
       }
 
       // and include anything mentioned in receipt logs
-      for (const { address } of receipt.logs) {
-        addresses.push(address);
+      for (const { address } of receipt?.logs ?? []) {
+        addresses.push(address as string);
```

Comment:
What was the type of address? It should be a string by default (@spacesailor24)

Reply:
The type of the `address` is `string | undefined`.